internal/pkg/encryption/api.go

```diff
@@ -12,9 +12,10 @@ import (
 )
 
 type Api struct {
-	hostUrl string
-	client  *client.Client
-	logger  *zap.SugaredLogger
+	hostUrl   string
+	projectId int
```

Comment:
Encryption API sifruje data pre konkretny projekt.
Doteraz sme `projectId` zadavali az pri sifrovani, ale to komplikuje zavislosti, lebo `projectId` mame ulozeny v strukture `StorageAPI`.
Preto `projectId` ulozime aj do `encryption.Api`, ... kedze vzdy pracujeme len nad jednym projektom.